blocks/product-comparison-table/product-comparison-table.css

```diff
@@ -0,0 +1,256 @@
+    padding-bottom: 0.4375rem;
+}
+
+.product-comparison-header div[role="columnheader"].active p.button-container a {
+    background-color: var(--button-background-color);
+    border: 2px solid var(--button-background-color);
+    color: var(--button-link-color);
+}
+
+.product-comparison-header div[role="columnheader"].active p.button-container a.button:hover::after {
+    background-color: var(--button-link-color);
+}
+
+.product-comparison-header div[role="columnheader"] h4 {
+    margin: 3rem 0.625rem 0.25rem 0.625rem;
+    font-size: var(--heading-font-size-s);
+}
+
+.product-comparison-header div[role="columnheader"] .price-strike-through {
+    text-decoration: line-through;
+}
+
+.product-comparison-header div[role="columnheader"] .old-price-container p{
+    font-size: var(--body-font-size-xxs);
+    line-height: 1.2;
+    color: var(--product-comparison-text-color);
+}
+.product-comparison-header div[role="columnheader"] .product-comparison-price {
+    padding-top: 1.875rem;
+}
+
+.product-comparison-header div[role="columnheader"] .product-comparison-price .current-price-container {
+    padding-top: 0.375rem;
+}
+
+.product-comparison-header div[role="columnheader"] .per-year-statement {
+    font-size: var(--body-font-size-xs);
+    color: var(--product-comparison-text-color);
+    padding-bottom: 0.9375rem;
+}
+
+.product-comparison-header div[role="columnheader"] .current-price-container p{
+    font-size: var(--body-font-size-m);
+    font-weight: var(--font-weight-boldest);
+}
+
+.product-comparison-table div[role='row']:not(.product-comparison-header) {
+    display: flex;
+    flex-wrap: wrap;
+    font-size: 14px;
+    font-family: var(--body-font-family);
+    color: #616161;
+    width: 100%;
+}
+
+.product-comparison-table div[role='columnheader'] {
+    display: table-cell;
+    text-align: center;
+}
+.product-comparison-table div[role='cell'] {
+    text-align: center;
+    vertical-align: middle;
+    background-color: var(--table-cell-color);
+    border-right: 1px solid #fff;
+    border-bottom: 1px solid #fff;
+    text-transform: capitalize;
+}
+
+.product-comparison-table div[role='cell']:first-child {
+    text-align: left;
+    background-color: var(--first-table-cell-color);
+    padding: 0.875rem 1.75rem 1.0625rem 2.125rem;
+    width: 100%;
+    font-weight: var(--font-weight-bold);
+    text-align: center;
+}
+
+.product-comparison-table div[role='cell']:not(:first-child) {
+    padding: 23px 10px;
+    flex: 1;
+    width: 168px;
+}
+
+.product-comparison-table div[role='cell'].active {
+    background: #fff;
+    box-shadow: 0 3px 8px 0 rgba(0,0,0,.1);
+    border-bottom: 1px solid #e4f2ff;
+    border-right: 1px solid #e4f2ff;
+}
+
+@media (min-width: 480px) {
+    .product-comparison-table {
+        width: unset;
+    }
+}
+
+@media (min-width: 990px) {
+    .product-comparison-table-container {
+        padding-left: 2rem !important;
```

Comment:
I put `!important` because on the mobile view is a different padding than the usual, and when going to a desktop size, the padding was not set back automatically

Reply:
found a fix for it